bin/iotagent-json

```diff
@@ -25,11 +25,8 @@
 'use strict';
 
 var iotAgent = require('../lib/iotagent-json'),
-    info = require('../package.json'),
-    context = {
```

Comment:
If debug could be left here, the context would be useful as it would refer specifically to this Agent